api/lib/domain/errors.js

```diff
@@ -138,6 +138,12 @@ class NoStagesForCampaign extends DomainError {
   }
 }
 
+class AccountRecoveryDemandExpired extends DomainError {
+  constructor(message = 'The account recovery demand has expired.') {
```

Comment:
** chipotage**
`constructor(message = 'This account recovery demand has expired.')`